### PR TITLE
Updated Home Screen Branding into WH Brand

### PR DESCRIPTION
**Pull request by ericjiang97**
Makes the home screen more pretty with the WH Branding 🔥🖌️

## Changes

- Branded Home Page with the Western Health assets
- Added transition between the scroll view and the main landing view.

## Screenshots

https://d.pr/v/UgSYfK

## Things to note
- Changed HomePage into a stateful widget from stateless to monitor scroll location to help transition between the transparent appbar into the white appbar, with the icon changing color as well.

### CODE REVIEW

**Comment by yangjinluo** [comment]
hi

**Comment by lukesleeman** [comment]
It looks amazing!

File: lib/utils/color.dart [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import 'package:flutter/material.dart';
+
+abstract class ColorUtils {
```
Comment (domesticmouse) [inline lib/utils/color.dart]: nit: drop the abstract class, it's not required. Just make the method a top level function.

File: lib/utils/color.dart [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  static Color generateIconColor(
+      Color fromColor, Color toColor, double deltaPercentage) {
+    var fromRed = fromColor.red,
```
Comment (domesticmouse) [inline lib/utils/color.dart]: final?

File: lib/view/home_page.dart [hunk 3]
```diff
@@ -1,68 +1,129 @@
...
+          cards: icu,
+        ),
+        // Make sure the bottom CardContainer has room to breathe.
+        SizedBox(height: 12),
+      ]),
+    );
+  }
+
+  Widget _renderBody(BuildContext context) {
+    var appBarHeight = 70, logoHeight = 24;
```
Comment (domesticmouse) [inline lib/view/home_page.dart]: final instead of var, put each variable on a different line.

File: lib/view/home_page.dart [hunk 4]
```diff
@@ -1,68 +1,129 @@
...
+                              AppColors.appBarIcon,
+                              percentage)),
                       onPressed: () => InfoView.navigateTo(context),
                     )
                   ],
                 ),
               ),
             ),
-            backgroundColor: AppColors.appBarBackground,
+            backgroundColor: Color.fromRGBO(255, 255, 255, percentage),
```
Comment (domesticmouse) [inline lib/view/home_page.dart]: Move this magic color to the AppStyle class as a method?

File: pubspec.lock [hunk 5]
```diff
@@ -328,7 +328,7 @@ packages:
       name: test_api
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "0.2.15"
+    version: "0.2.11"
```
Comment (domesticmouse) [inline pubspec.lock]: I suspect this means you aren't on beta yet. You'll get errors when I land my change. See [here](https://github.com/flutter/flutter/wiki/Flutter-build-release-channels) for more detail. In short:

```
$ flutter channel beta
$ flutter upgrade
```

And then re-start your editor / IDE.

**Review by domesticmouse — APPROVED** [review]

File: lib/utils/color.dart (hunk 1 above)
Comment (ericjiang97) [reply lib/utils/color.dart]: Pulled function into top-level :)

File: lib/utils/color.dart (hunk 2 above)
Comment (ericjiang97) [reply lib/utils/color.dart]: yep! and applied to rest

File: lib/view/home_page.dart (hunk 3 above)
Comment (ericjiang97) [reply lib/view/home_page.dart]: yep! and applied to rest

File: lib/view/home_page.dart (hunk 4 above)
Comment (ericjiang97) [reply lib/view/home_page.dart]: yep!

File: pubspec.lock (hunk 5 above)
Comment (ericjiang97) [reply pubspec.lock]: Done! Looks like its working (and this lock file is updated)

**Comment by maks** [comment]
> @ericjiang97 I think you just need to fix conflicts with latest master.

Yep I'll do that when I jump on. ☺

**Comment by ericjiang97** [comment]
@maks Updated!

**Comment by lukesleeman** [comment]
👋 Hey @ericjiang97 I've had a look at whats going on with this PR.  It seems that flutter analyzer is failing with the following:

```
   info • Avoid defining a class that contains only static members • lib/style.dart:4:1 • avoid_classes_with_only_static_members
   info • Prefer const over final for declarations • lib/style.dart:17:10 • prefer_const_declarations
   info • Prefer relative imports for files in `lib/` • lib/view/home_page.dart:4:8 • prefer_relative_imports
   info • Prefer final for variable declarations if they are not reassigned • lib/view/home_page.dart:19:9 • prefer_final_locals
   info • Prefer using /// for doc comments • lib/view/home_page.dart:28:3 • slash_for_doc_comments
   info • Prefer const with constant constructors • lib/view/home_page.dart:57:9 • prefer_const_constructors
   info • Prefer const over final for declarations • lib/view/home_page.dart:63:5 • prefer_const_declarations
   info • Prefer const over final for declarations • lib/view/home_page.dart:64:5 • prefer_const_declarations
```

here is a link: https://codemagic.io/app/5e75f4bfdd1d6d000a198e90/build/5e7c92e629657a000e2e3ebc

I don't know why that's not getting reported up to the build status here, and its just showing "Some checks haven't completed"

You can check on the build statuses via the `#covid-19-app-builds` channel in our slack.  Thats where I saw these errors.

**Comment by ericjiang97** [comment]
Resolved #48

**Pull request merged**